installers/msi-language/StateDeploy/CustomAction.cs

```diff
@@ -55,11 +55,11 @@ public static ActionResult InstallStateTool(Session session, out string stateToo
                 return ActionResult.Failure;
             }
 
-            string installCmd = string.Format("Set-PSDebug -trace 2; Set-ExecutionPolicy Unrestricted -Scope Process; \"{0}\" -n -t \"{1}\"", scriptPath, installPath);
+            string installCmd = string.Format("Set-PSDebug -trace 2; Set-ExecutionPolicy Unrestricted -Scope Process; & '{0}' -n -t \"\"\"{1}\"\"\"", scriptPath, installPath);
```

Comment:
Couldn't we single quote the `installPath` as well?

Reply:
You would think so...  I did too, tried it and got an error... :eyes: